ui_testing/testcases/basic_tests/test06_orders.py

```diff
@@ -958,32 +978,35 @@ def test022_create_existing_order_with_test_units_and_change_material_type(self)
             test_units_list.append(test_unit_dict['Test Unit Name'])
 
         self.order_page.get_orders_page()
-        created_order = self.order_page.create_new_order(material_type='r', article='a', contact='a',
-                                                         test_units=test_units_list)
-
-        created_existing_order = self.order_page.create_existing_order_with_auto_fill(no=created_order.replace("'", ""))
+        created_order = self.order_page.create_order_with_test_unit(material_type='r', article='a', contact='a',
```

Comment:
why you update This test case in this PR, while travis file contains only LIMS-3406

Reply:
Done